### ISSUE TITLE
TabContainer doesnt show content of second tab

### ISSUE THREAD

**Issue by cbluth** [fyne-io/fyne#536]
**Describe the bug**
The content of the 2nd tab is missing:
![image](https://user-images.githubusercontent.com/6709636/68998256-7e08e580-08b0-11ea-9ec8-110f08ddd8d4.png)
![image](https://user-images.githubusercontent.com/6709636/68998262-86f9b700-08b0-11ea-8ad4-1168b0ac0174.png)

**To Reproduce**
1. Run the code seen here: https://tour.fyne.io/widget/tabcontainer.html

here is the code:
```
package main

import (
	"fyne.io/fyne/app"
	//"fyne.io/fyne/theme"
	"fyne.io/fyne/widget"
)

func main() {
	myApp := app.New()
	myWindow := myApp.NewWindow("TabContainer Widget")

	tabs := widget.NewTabContainer(
		widget.NewTabItem("Tab 1", widget.NewLabel("Hello")),
		widget.NewTabItem("Tab 2", widget.NewLabel("World!")))

	//widget.NewTabItemWithIcon("Home", theme.HomeIcon(), widget.NewLabel("Home tab"))

	tabs.SetTabLocation(widget.TabLocationLeading)

	myWindow.SetContent(tabs)
	myWindow.ShowAndRun()
}
```

**Device (please complete the following information):**
 - OS: Ubuntu 19.10
 - Go version: go version go1.13.3 linux/amd64
 - Fyne version:
fyne.io/fyne v1.1.2 h1:a9YLFXxqN7lKNqTrk+ocw3/3ROrn6aFiofix8ATVOBc=
fyne.io/fyne v1.1.2/go.mod h1:ukZVsvlHO29NK4HARH2B4rqgVqgx6WbtWKfEUOz4ZNo=

**Comment by andydotxyz** [comment]
Thanks for the report. Does the content appear if you show the tab a second time?

**Comment by AndrewWPhillips** [comment]
I reproduced the problem (Windows 10) and I think I tracked it down and a (possible) solution.

When `Label.Show()` is called for the 2nd tab, `canvas.Refresh()` is called for the label but it is not in the gl.canvases map so CanvasForObject returns nil, hence 'c' is nil (at base.go line 70) and the label is not refreshed.  I think this is because the textCanvas within the label in the second tab is hidden.

The fix that seems to work is to change line 69 of label.go from:
```
	l.textProvider.Hidden = hidden
```
to:
```
	l.textProvider.Hidden = false
```
However, I am not sure if this has other consequence so I did not submit a pull request.
